pandas/core/frame.py

```diff
@@ -5980,6 +5980,49 @@ def melt(self, id_vars=None, value_vars=None, var_name=None,
                     var_name=var_name, value_name=value_name,
                     col_level=col_level)
 
+    def explode(self, col_name, sep=None, dtype=None):
+        """
+        Create a new DataFrame where each element in each row
```

Comment:
I think this has to be a single line.

Then you can have a multi-line extended summary. `scripts/validate_docstrings pandas.core.frame.explode` should print out all the issues.

Reply:
ah ok. will fix. thanks